File: src/main/java/kr/co/lunatalk/domain/member/repository/MemberRepositoryCustom.java
package kr.co.lunatalk.domain.member.repository;

import kr.co.lunatalk.domain.member.domain.Member;
import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;

import java.util.List;

public interface MemberRepositoryCustom {
	Page<Member> findMembers(Pageable pageable);
}

File: lunatalk-core/src/main/kotlin/kr/co/lunatalk/domain/member/repository/MemberRepositoryCustom.kt
package kr.co.lunatalk.domain.member.repository

import kr.co.lunatalk.domain.member.domain.Member
import org.springframework.data.domain.Page
import org.springframework.data.domain.Pageable

interface MemberRepositoryCustom {
    fun findMembers(pageable: Pageable): Page<Member>
}
